client/src/pages/Project.tsx

```diff
@@ -173,6 +240,293 @@ const Project = (): JSX.Element => {
         );
     }
 
+    const handleChangeName: React.MouseEventHandler<HTMLButtonElement> = (event) => {
```

Comment:
handle change should probably either close the modal or at least clear the field (or some other feedback that it works?) -> same for all the other changes

Reply:
hmm I get what you mean, will add a confirmation message on top of settings page based on success or failure